front/public/locales/fr/stdcm.json

```
@@ -114,6 +114,18 @@
   "stdcmErrors": {
     "bothPointAreScheduled": "Le calcul ne peut pas prendre en compte à la fois un horaire d'origine et un horaire de destination. Il faut choisir l'un ou l'autre :",
     "infraNotLoaded": "Infrastructure en cours de chargement",
+    "missingFields": {
+      "departureTime": "L'horaire de départ",
+      "destination": "La destination",
+      "origin": "L'origine",
+      "timeDestination": "l'heure de départ à la destination",
```

I guess ? (same in english)
```suggestion
      "timeDestination": "l'heure d'arrivée à la destination",
```

---

done